vivarium/core/experiment.py

```diff
@@ -805,7 +806,11 @@ def schema_topology(self, schema, topology):
                     if path is None:
                         path = (key,)
                     node = self.get_path(path)
-                    state[key] = node.schema_topology(subschema, {})
+                    if node:
+                        state[key] = node.schema_topology(subschema, {})
```

Comment:
I think we still want to set the state key here since it was requested in the topology even though the value is `None`, in order to mirror the structure of the topology passed in. `None` is a legitimate value.

Reply:
In the case I found ```node``` was equal to ```None```, which can't have ```schema_topology``` called on it.